honeybadgermpc/hbavss_light.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import asyncio
+from collections import defaultdict
+from pickle import dumps, loads
+from honeybadgermpc.betterpairing import ZR
+from honeybadgermpc.polynomial import polynomials_over
+from honeybadgermpc.poly_commit import PolyCommit
+from honeybadgermpc.symmetric_crypto import SymmetricCrypto
+from honeybadgermpc.exceptions import HoneyBadgerMPCError
+from honeybadgermpc.protocols.reliablebroadcast import reliablebroadcast
+
+
+class HbAVSSMessageType:
+    OK = "OK"
+
+
+class HbAvss(object):
+    def __init__(self, public_keys, private_key, g, h, n, t, my_id, send, recv):
+        self.public_keys, self.private_key = public_keys, private_key
+        self.n, self.t, self.my_id = n, t, my_id
+        self.g = g
+        self.poly_commit = PolyCommit(g, h)
+        # A different set of send, recv values must be passed for each AVSS object.
+        # If the same sends, recvs are reused then the recv_loops of the two objects
+        # can get values belonging to the other object.
+        self.send, self.recv = send, recv
+        self.field = ZR
+        self.poly = polynomials_over(self.field)
+
+        # When running multiple AVSSes, the messages are forwarded to the correct
+        # AVSS using the queue. The key is the AVSS ID which is sent along with
+        # each AVSS message.
+        self.queues = defaultdict(asyncio.Queue)
```

Comment:
So a single instance of this `HbAvss` object supports multiple AVSS's from the same dealer, right?

Reply:
Yes, this is demonstrated in the tests.